### PR TITLE
Nc news 5

### PR DESCRIPTION
**Pull request by YShamm**
Task 6

### CODE REVIEW

**Review by Dutton1990 — CHANGES_REQUESTED** [review]
Great job Yahya just one more test needed! Although with your implementation it might pass straight away. 

You need to check that for an article that exists but has no comments yet - that we get a 200 back and an empty array. 

Article 2 I believe has no comments on it yet. 

You have implemented a promise.all to check that the article exists, so I think this might work without much more tinkering. Please add that test, check that it passes and then push again to this branch. Re-send the PR and one of us will check it before merge 😄

File: __tests__/app.test.js
Comment (Harrison365) [inline __tests__/app.test.js:176]: Great addition ^^^

Well done 👍

**Comment by Harrison365** [comment]
Good to merge when ready ✅ 
Well done Yahya 👍

**Pull request merged**
